### PR TITLE
More StatementList tests

### PR DESCRIPTION
**Pull request by thiemowmde**
This is the opposite of what we discussed in #453.

I also moved the private getters to the top.

### CODE REVIEW

File: tests/unit/Statement/StatementListTest.php [hunk 1]
```diff
@@ -17,9 +17,66 @@
...
 class StatementListTest extends \PHPUnit_Framework_TestCase {
 
+	/**
+	 * @param int $propertyId
+	 * @param string|null $guid
+	 * @param int $rank
+	 *
+	 * @return Statement
+	 */
+	private function getStatement( $propertyId, $guid, $rank = Statement::RANK_NORMAL ) {
```
Comment (Benestar) [inline tests/unit/Statement/StatementListTest.php]: `getStatementMock` perhaps?

**Comment by Benestar** [comment]
+1 Yey more tests

File: tests/unit/Statement/StatementListTest.php (hunk 1 above)
Comment (thiemowmde) [inline tests/unit/Statement/StatementListTest.php]: Personally I really dislike the `get...Mock` naming scheme. Code that calls this should not care if it gets a mock or a real object. As the `@return` tag states.

Comment (JeroenDeDauw) [inline tests/unit/Statement/StatementListTest.php]: Success. Brainwashing of @thiemowmde complete :D

**Comment by JeroenDeDauw** [comment]
All fine. Found this misleading though:

> I also moved the private getters to the top.

This made me think about actual getters in the production code. The things moved in the test are not getters.

**Comment by thiemowmde** [comment]
I disagree. `get` is not a reserved word that can only have a single, well defined meaning. Also: unrelated to this pull request.

**Comment by JeroenDeDauw** [comment]
`getter` has a well defined meaning

**Comment by thiemowmde** [comment]
It doesn't say where the object the caller gets is coming from, so no, calling a get method a getter is not "misleading". Still unrelated to this pull request.

**Pull request merged**